string picker_id
float64 picking_rate
float64 transportation_rate
float64 tray_capacity
float64 picking_progress
int32 n_trays
int32 tot_trays
int32 n_rows
string curr_row

